docs/source/internal/generation_utils.rst

```diff
@@ -13,7 +13,7 @@
 Utilities for Generation
 -----------------------------------------------------------------------------------------------------------------------
 
-This page lists all the utility functions used by :meth:`~transformers.PreTrainedModel.generate`,
+This page lists all the utility functions used by :meth:`~transformers.generation_utils.GenerationMixin.generate`,
 :meth:`~transformers.PreTrainedModel.greedy_search`, :meth:`~transformers.PreTrainedModel.sample`,
 :meth:`~transformers.PreTrainedModel.beam_search`, :meth:`~transformers.PreTrainedModel.beam_sample`, and
 :meth:`~transformers.PreTrainedModel.group_beam_search`.
```

Comment:
I think those should point to the GenerationMixin as well, as it's the place where they come from and they are probably not resolved either.